CVLByExamples/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankFixed.conf

```diff
@@ -0,0 +1,16 @@
+{
+    "files": [
+        "contracts/VulnerableBankFixed.sol"
+    ],
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "process": "emv",
+    "solc": "solc8.17",
+    "disable_auto_cache_key_gen": true,
+    "server": "production",
+    "prover_version": "master",
+    "verify": "VulnerableBankFixed:certora/spec/ReadOnlyReentrancy.spec",
+    "prover_args": [
+         "-ignoreViewFunctions"
+    ]
```

Comment:
must?

Reply:
It is faster with it and the view functions don't appear in the report this way. They should not be processed.

Reply:
okay